src/Core/src/Platform/ViewExtensions.cs

```diff
@@ -100,6 +115,17 @@ public static partial class ViewExtensions
 		}
 #endif
 
+		internal static bool IsThisMyPlatformView(this IElement? element, PlatformView platformView)
```

Comment:
I don't love the name of this. SelfPlatformView? I don't know what's better.

Reply:
I don't mind it. 
It seems fairly clear what you're asking from it.

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/14089/files#diff-3d7ab0c43b8b664d77eeae10a85e7f2c5eb8de0daefc274e0d3723dbbb8fe577R35 :-)